### PR TITLE
Configuration Mapping User Permissions for Edly Roles

### PR DESCRIPTION
**Pull request by asadmanzoor93**
This PR adds User groups mapping configurations of roles on Edly Panel, LMS and WordPress

Create the following groups in LMS Admin:
```
    Edly Panel Restricted Users
    Edly Panel Users
    Edly Panel Admin Users
    WordPress Subscriber Users
    WordPress Edly Admin Users
    Admin Configuration Users
```

### CODE REVIEW

File: lms/envs/common.py [hunk 1]
```diff
@@ -1327,9 +1327,29 @@ def _make_locale_paths(settings):
...
-EDLY_PANEL_GROUP_NAME = 'Edly Panel Users'
+EDLY_PANEL_RESTRICTED_USERS = 'Edly Panel Restricted Users'
+EDLY_PANEL_USERS_GROUP_NAME = 'Edly Panel Users'
+EDLY_PANEL_ADMIN_USERS_GROUP_NAME = 'Edly Panel Admin Users'
+WORDPRESS_SUBSCRIBER_USERS_GROUP_NAME = 'WordPress Subscriber Users'
+WORDPRESS_EDLY_ADMIN_USERS_GROUP_NAME = 'WordPress Edly Admin Users'
 ADMIN_CONFIGURATION_USERS_GROUP_NAME = 'Admin Configuration Users'
 
+EDLY_ROLES = {
+    'panel_restricted': EDLY_PANEL_RESTRICTED_USERS,
```
Comment (zubair-arbi) [inline lms/envs/common.py]:
```
edly_panel_users
edly_panel_admin_users
edly_panel_restricted_users
edly_wp_admin_users
edly_wp_subscriber_users
```

File: lms/envs/common.py [hunk 2]
```diff
@@ -1327,9 +1327,29 @@ def _make_locale_paths(settings):
...
 # Edly Configuration
-EDLY_PANEL_GROUP_NAME = 'Edly Panel Users'
+EDLY_PANEL_RESTRICTED_USERS = 'Edly Panel Restricted Users'
+EDLY_PANEL_USERS_GROUP_NAME = 'Edly Panel Users'
+EDLY_PANEL_ADMIN_USERS_GROUP_NAME = 'Edly Panel Admin Users'
+WORDPRESS_SUBSCRIBER_USERS_GROUP_NAME = 'WordPress Subscriber Users'
+WORDPRESS_EDLY_ADMIN_USERS_GROUP_NAME = 'WordPress Edly Admin Users'
 ADMIN_CONFIGURATION_USERS_GROUP_NAME = 'Admin Configuration Users'
 
+EDLY_ROLES = {
```
Comment (zubair-arbi) [inline lms/envs/common.py]: `EDLY_USER_ROLES`

File: lms/envs/common.py [hunk 3]
```diff
@@ -1327,9 +1327,29 @@ def _make_locale_paths(settings):
...
+    'panel_restricted': EDLY_PANEL_RESTRICTED_USERS,
+    'panel_user': EDLY_PANEL_USERS_GROUP_NAME,
+    'panel_admin': EDLY_PANEL_ADMIN_USERS_GROUP_NAME,
+    'subscriber': WORDPRESS_SUBSCRIBER_USERS_GROUP_NAME,
+    'edly_admin': WORDPRESS_EDLY_ADMIN_USERS_GROUP_NAME
+}
+
+EDLY_ROLES_LIST = [
```
Comment (zubair-arbi) [inline lms/envs/common.py]: Why do we need this list when we have `EDLY_USER_ROLES` dictionary above?

File: lms/envs/common.py [hunk 4]
```diff
@@ -1327,9 +1327,29 @@ def _make_locale_paths(settings):
 ]
 
 # Edly Configuration
-EDLY_PANEL_GROUP_NAME = 'Edly Panel Users'
+EDLY_PANEL_RESTRICTED_USERS = 'Edly Panel Restricted Users'
```
[the remark this answers is not in the document]
Comment (asadmanzoor93) [reply lms/envs/common.py]: Feedback Implemented

File: lms/envs/common.py (hunk 2 above)
Comment (asadmanzoor93) [reply lms/envs/common.py]: Feedback Implemented

File: lms/envs/common.py (hunk 3 above)
Comment (asadmanzoor93) [reply lms/envs/common.py]: This was duplicated, which is now code removed.

**Pull request merged**